### ISSUE TITLE
[Feature] Open game in tab

### ISSUE THREAD

**Issue by nodiaque** [rommapp/romm#1312]
Hello,

I just recently started using Romm and while it's great, there is something that's really bugging me. I imported all my roms and most have bad name thus no match. When trying to match them, sometime I need to start the rom to know what it is. Once I know what it is, I can match it through the interface but then, I have to come back to the library which is slow (too much rom I guess) and I need to scroll back to where I was.

Having the ability to set an option where all play button in the library gui have a target=new so it open a new tab instead would be nice. It could even open a tab that has no bar and nothing so it's just the game.

Having the option to set that also for other button would be nice.

Thank you

**Comment by mikhailv** [comment]
Hi, faced with similar issue. In my case I wanted to manually match unmatched games without clicking on every game, editing it and clicking back to get list of games and choose next one. So as workaround you may click to game card (image) with Ctrl pressed it will open game page in new tab. Doing so you may open multiple game pages and switch between them quickly without navigating back to game list.

Ctrl+click does not work on "play" button for me.

**Comment by zurdi15** [comment]
You can match the games from the gallery with the three dot menu without the need of clicking each one btw

**Comment by mikhailv** [comment]
Yes, but I wanted to see original filename (sometimes automatic match may match game to wrong number version). But later I found easier solution to switch game grid to list view, where file name is visible and use three-dot menu as you suggested. Thanks.

Btw, @zurdi15, thank you for RomM - it is really good tool to manage collection.

**Comment by nodiaque** [comment]
> You can match the games from the gallery with the three dot menu without the need of clicking each one btw

That's great when you know what game it is. I have named rom 00001,0002 etc so I have to launch the game first to know what it is, do a manual match and rename.

The fact that the list get reloaded everytime (and is very slow to do it even on my beast of server) make it a pain. And the fact it send me back to the beginning of the list, disregarding any filter I had.

**Comment by mikhailv** [comment]
Not sure, but maybe for you will be easier to find collection of roms with proper naming? Or as alternative use RetroArch or tools like https://igir.io/ to match game automatically by file hash/content, not by file name as RomM does.

**Comment by adamantike** [comment]
Regarding the specific "Open game in tab" feature, would it work if we change the Rom gallery, for some of the current interactions to also open when using middle (wheel) click? Like the "Play" button or accessing to the game information

**Comment by nodiaque** [comment]
It would be already a good change, just unsure how that would work with trackpad on laptop

**Comment by gantoine** [comment]
Given that the initial issue can be avoided in multiple ways (using the list view, middle clicking the play button), I'm going to close this issue as not planned. We'll be doing work on the gallery views in the upcoming months which should address some of your concerns.

**Comment by gantoine** [comment]
@nodiaque Would https://github.com/emmercm/igir help here? It has support for .DAT files so filenames shouldn't be an issue.

**Comment by nodiaque** [comment]
. Dat file? Unsure what these are for. It's my rom file that have bad name that I need to launch in order to know what they are before renaming

**Comment by gantoine** [comment]
Check out the igir docs, I think they'll help you a ton with what you're trying to fix! https://igir.io/
